src/libsyntax/feature_gate.rs

```diff
@@ -554,6 +554,9 @@ declare_features! (
     // Allows using C-variadics.
     (active, c_variadic, "1.34.0", Some(44930), None),
 
+    // Allows `if/while p && let q = r && ...` chains.
+    (active, let_chains, "1.36.0", Some(53667), None),
```

Comment:
Could you change the feature name to `let_expressions` or something, to make it closer to what is implemented and to make people aware of it.

Reply:
I would prefer to keep this as-is because while `let` expressions are syntactically supported they are not semantic expressions, which is more important imo. So I think `let_expressions` would be misleading.